src/test/java/org/sagebionetworks/web/unitclient/widget/table/v2/results/cell/EnumFormCellEditorImplTest.java

```diff
@@ -5,71 +5,121 @@
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mockito;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.sagebionetworks.web.client.PortalGinInjector;
+import org.sagebionetworks.web.client.view.DivView;
 import org.sagebionetworks.web.client.widget.table.v2.results.cell.EnumFormCellEditor;
+import org.sagebionetworks.web.client.widget.table.v2.results.cell.ListCellEditorView;
 import org.sagebionetworks.web.client.widget.table.v2.results.cell.RadioCellEditorView;
 
+@RunWith(MockitoJUnitRunner.class)
 public class EnumFormCellEditorImplTest {
 
-	RadioCellEditorView mockView;
+	@Mock
+	DivView mockDivView;
+	@Mock
+	PortalGinInjector mockGinInjector;
+	@Mock
+	ListCellEditorView mockListView;
+	@Mock
+	RadioCellEditorView mockRadioView;
 	EnumFormCellEditor editor;
 
 	@Before
 	public void before() {
-		mockView = Mockito.mock(RadioCellEditorView.class);
-		editor = new EnumFormCellEditor(mockView);
+		when(mockGinInjector.createListCellEditorView()).thenReturn(mockListView);
+		when(mockGinInjector.createRadioCellEditorView()).thenReturn(mockRadioView);
+		editor = new EnumFormCellEditor(mockDivView, mockGinInjector);
 	}
 
+	private List<String> getValuesForDropdownList() {
```

Comment:
Helper method, generates MAX_RADIO_BUTTONS + 1 values (to trigger the dropdown list renderer)